src/db/couchApiKeys.ts

```diff
@@ -46,6 +46,8 @@ export async function getApiKeyByKey(
   apiKey: string
 ): Promise<ApiKey | undefined> {
   const db = connection.db.use(couchApiKeysSetup.name)
+  if (apiKey === '') return
+
```

Comment:
Why don't we want to return 500? How would the client know whether this operation has succeeded?

Reply:
We return `undefined` at the database layer, since the database is also used by the CLI / price scripts / etc. Then the `withApiKey` middleware turns it into an HTTP response.